closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -61,12 +63,17 @@ endif # OPENJCEPLUS_JGSKIT_PLATFORM
 compile-libs :
 	@$(ECHO) Compiling OpenJCEPlus native code
 	export \
-			$(OPENJCEPLUS_VS_LIB) \
 			GSKIT_HOME=$(OPENJCEPLUS_GSKIT_HOME) \
-			JAVA_HOME=$(OPENJCEPLUS_BOOT_JDK) \
+			JAVA_HOME=$(OPENJCEPLUS_JDK) \
 			JCE_CLASSPATH=$(OPENJCEPLUS_JCE_CLASSPATH) \
+			$(OPENJCEPLUS_VS_LIB) \
 			PLATFORM=$(OPENJCEPLUS_JGSKIT_PLATFORM) \
```

Comment:
Suggest rather than exporting these variables, they should be appear on the `make` command line (like `OPENJCEPLUS_HEADER_FILES`).

Reply:
Moved required variables to `make` command instead of exporting.